docs/revsets.md

```diff
@@ -78,6 +78,144 @@ only symbols.
 You can use parentheses to control evaluation order, such as `(x & y) | z` or
 `x & (y | z)`.
 
+<details>
+<summary>Examples:</summary>
+
+Given this history:
+```
+D
+|\
+| o C
+| |
+o | B
+|/
+o A
+|
+o root()
+```
+
+**Operator** `x-`
+
+`D-` ⇒ `{C,B}`
+
```

Comment:
Unfortunately, this did not work out well in https://martinvonz.github.io/jj/prerelease/revsets/. I'm not sure what to do about it, that is what exactly does and does not work inside `<details>` in MkDocs and whether there are extensions that can make it better.

Reply:
One option would be to  use an examples callout that is not collapsible. 

We can try https://squidfunk.github.io/mkdocs-material/reference/admonitions/#collapsible-blocks.

Here's a fancy option: https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#embedded-content

Reply:
This was fixed in #3911.